### PR TITLE
Add kubetest flag defining how long it wait on a new Boskos resource

### PR DESCRIPTION
**Pull request by chaodaiG**
Currently kubetest waits on getting a Boskos in a predefined 5 minutes window, this PR adds a flag to Kubetest to allow it taking a parameter overriding this timeout. This can help alleviate Boskos running out of resources problem when there is a sharp spike of Boskos resources usage

/cc @cjwagner 
/cc @krzyzacy

### CODE REVIEW

File: kubetest/main.go [hunk 1]
```diff
@@ -181,6 +182,7 @@ func defineFlags() *options {
 	flag.DurationVar(&timeout, "timeout", time.Duration(0), "Terminate testing after the timeout duration (s/m/h)")
 	flag.BoolVar(&o.up, "up", false, "If true, start the e2e cluster. If cluster is already up, recreate it.")
 	flag.StringVar(&o.upgradeArgs, "upgrade_args", "", "If set, run upgrade tests before other tests")
+	flag.IntVar(&o.boskosWaitDuration, "boskos-wait-duration", 5, "Defines how long it waits until quite getting Boskos resoure, default 5 minutes")
```
Comment (krzyzacy) [inline kubetest/main.go]: can you make it a DurationVar instead? :-)

**Comment by cjwagner** [comment]
I'll let Sen review this one.
/uncc
/ok-to-test

File: kubetest/main.go (hunk 1 above)
Comment (krzyzacy) [reply kubetest/main.go]: (also, s/quite/quit?)

Comment (chaodaiG) [reply kubetest/main.go]: done, both. Good catch!

**Pull request merged**